tailwind.config.js

```diff
@@ -15,8 +18,18 @@ module.exports = {
         "2xl": "1400px",
       },
     },
+    screens: {
+      xs: "425px",
+      ...defaultTheme.screens,
+    },
```

Comment:
dodaj do extend to nie bedzie nadpisywac

Reply:
https://tailwindcss.com/docs/screens#adding-smaller-breakpoints

Reply:
spoko